packages/recorder-loader/src/loader.types.ts

```diff
@@ -1,7 +1,12 @@
 import { NetworkResponseSanitizer } from "@alwaysmeticulous/sdk-bundles-api";
 
 export interface LoaderOptions {
-  projectId: string;
+  /**
+   * @deprecated Renamed to recordingToken. Please use the same value but pass it as the recordingToken instead.
+   */
+  projectId?: string;
+  recordingToken: string;
```

Comment:
This will be a breaking change but I guess that's okay?

Reply:
Yeah I think it's OK because it should be very clear given the deprecation message and it's only when you bump the NPM dep